repo/migrations/Migration011.go

```diff
@@ -0,0 +1,190 @@
+package migrations
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/OpenBazaar/wallet-interface"
+	"path"
+	"strings"
+)
+
+var WalletCoinType wallet.CoinType
```

Comment:
We have been prefixing public vars in `migrations` with `MigrationXXX_ConstantName` pattern so we can avoid colliding vars inside this package. Ex: https://github.com/OpenBazaar/openbazaar-go/blob/master/repo/migrations/Migration009.go#L17